test/common/stats/utility_fuzz_test.cc

```diff
@@ -16,14 +16,21 @@ DEFINE_FUZZER(const uint8_t* buf, size_t len) {
     const absl::string_view string_buffer(reinterpret_cast<const char*>(buf), len);
     Stats::Utility::sanitizeStatsName(string_buffer);
   }
+  if (len < 1)
+    return;
 
   // generate a random number as the maximum length of the stat name
-  const size_t max_len = *reinterpret_cast<const uint8_t*>(buf) % len;
+  const size_t max_len = 1 + *reinterpret_cast<const uint8_t*>(buf) % len;
```

Comment:
s/% len/% (len -1)/ I think.

But looking at the logic below you are actually going to be consuming a bunch of random-length strings. Do you want to to limit yourself to something more like 10% or 25% of the available bytes so that the names below are likely non-empty?

You do the exact math for that, based on the number of control-bytes you are pulling out, plus the number of random-length strings, all of which have to fit in `len`